frontend/components/person-card/index.js

```diff
@@ -47,16 +47,22 @@ const PersonCard = ({
     triggerOnce: true
   });
   const isMobile = useMedia(['(min-width: 1025px)'], [false], true);
-
+  let mentor = false;
+  if (sub_title == 'Mentor') {
+    mentor = true;
+  } else {
+    mentor = false;
+  }
```

Comment:
For code readability purposes, I suggest using a function to wrap the logic that calculates the value of `mentor` and m `mentee`.